test/cluster/mzcompose.py

```diff
@@ -113,120 +113,86 @@ def workflow_test_smoke(c: Composition, parser: WorkflowArgumentParser) -> None:
     args = parser.parse_args()
 
     c.down(destroy_volumes=True)
-    c.up("zookeeper", "kafka", "schema-registry", "localstack")
-    c.up("materialized")
-
-    # Create a cluster and verify that tests pass.
-    c.up("clusterd1")
-    c.up("clusterd2")
-
-    # Make sure cluster1 is owned by the system so it doesn't get dropped
-    # between testdrive runs.
-    c.sql(
-        """
-        ALTER SYSTEM SET unsafe_enable_unorchestrated_cluster_replicas = true;
-
-        DROP CLUSTER IF EXISTS cluster1 CASCADE;
-
-        CREATE CLUSTER cluster1 REPLICAS (
-            replica1 (
-                STORAGECTL ADDRESSES ['clusterd1:2100', 'clusterd2:2100'],
-                STORAGE ADDRESSES ['clusterd1:2103', 'clusterd2:2103'],
-                COMPUTECTL ADDRESSES ['clusterd1:2101', 'clusterd2:2101'],
-                COMPUTE ADDRESSES ['clusterd1:2102', 'clusterd2:2102'],
-                WORKERS 2
-            )
-        );
 
-        GRANT ALL ON CLUSTER cluster1 TO materialize;
-        """,
-        port=6877,
-        user="mz_system",
-    )
+    with c.override(
```

Comment:
Most of the diffs in this file come from adding overrides, which force indentation changes. I recommend reviewing with whitespace changes disabled!